arch/arm/boot/dts/overlays/README

```diff
@@ -5249,6 +5249,7 @@ Params: sizex                   Touchscreen size x (default 720)
         invy                    Touchscreen inverted y axis
         swapxy                  Touchscreen swapped x y axis
         disable_touch           Disables the touch screen overlay driver
+        rotatation              Display rotation {0,90,180,270} (default 0)
```

Comment:
As the dtoverlaycheck docs noticed, typo in `rotatation`

Reply:
Corrected.